PlaywrigthSpecFlow/Features/Account/Account.feature

```diff
@@ -0,0 +1,8 @@
+﻿@ICreateAccountByAPI
+@IDeleteAccountByAPI
+Feature: Account
+
+Add account and verify status
+Scenario: Add a new account
+	Then I get success status code from API
+	Then I get message that account deleted after cleanup
```

Comment:
remove this step as it is can't check anything AFTER cleanup since cleanup happens AFTER all scenarios are executed (not just this scenario, but all scenarios in this feature if we had more than one)